### PR TITLE
Update case-cond-and-if.md

### PR DESCRIPTION
**Pull request by Himanshu21git**
Added correct link for pattern and guards documentation.

### CODE REVIEW

**Comment by josevalim** [comment]
Where is the link broken?

File: lib/elixir/pages/getting-started/case-cond-and-if.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ iex> case :ok do
 ** (CaseClauseError) no case clause matching: :ok
 ```
 
-The documentation for the `Kernel` module lists all available guards in its sidebar. You can also consult the complete [Patterns and Guards](../references/patterns-and-guards.html#guards) reference for in-depth documentation.
+The documentation for the `Kernel` module lists all available guards in its sidebar. You can also consult the complete [Patterns and Guards](patterns-and-guards.html#guards) reference for in-depth documentation.
```
Comment (josevalim) [inline lib/elixir/pages/getting-started/case-cond-and-if.md:68]:
```suggestion
The documentation for the `Kernel` module lists all available guards in its sidebar. You can also consult the complete [Patterns and Guards](../references/patterns-and-guards.md#guards) reference for in-depth documentation.
```

**Comment by josevalim** [comment]
Nvm, I see it!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Comment by Himanshu21git** [comment]
Has the documentation been revised to reflect the changes in version 1.16.0-rc.0?

**Comment by michallepicki** [comment]
@Himanshu21git this change will be visible in documentation of the next RC version

(btw it should be probably also cherry-picked onto the main branch)

**Pull request merged**